microsoft-edge/devtools-guide-chromium/sources/index.md

```diff
@@ -475,7 +470,7 @@ The following subsections cover debugging:
 <!-- ------------------------------ -->
 #### The basic approach to using a debugger
 
```

Comment:
todo 4: update https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/sources/index-images/sources-paused-breakpoint-highlights.png which shows Activity Bar on left.

rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/sources/?branch=pr-en-us-2717#the-basic-approach-to-using-a-debugger